source/docs/zero-to-robot/step-3/radio-programming.rst

```diff
@@ -206,6 +206,9 @@ If an error occurs during the configuration process, follow the instructions in
 Troubleshooting
 ---------------
 
-- :ref:`Disable all other network adapters <docs/networking/networking-introduction/roborio-network-troubleshooting:Disabling Network Adapters>`
-- Make sure you wait long enough the power light has stayed solid for 10 seconds.
-- Make sure you have the correct network interface.
+- :ref:`Disable all other network adapters <docs/networking/networking-introduction/roborio-network-troubleshooting:Disabling Network Adapters>`.
+- Make sure you wait long enough that the power light has stayed solid for 10 seconds.
```

Comment:
I think technically it should be "stays" but I also think this sounds better:
Wait until the power light has been solid for at least 10 seconds.